Tasks/NpmV1/npmpublish.ts

```diff
@@ -47,4 +48,67 @@ export async function getPublishRegistry(packagingLocation: PackagingLocation):
             break;
     }
     return npmRegistry;
+}
+
+async function getNpmRegistry(defaultPackagingUri: string, feed: any, authOnly?: boolean, useSession?: boolean) {
```

Comment:
Can you do better than `any` for `feed`'s type?

Reply:
Looks like the function itself in the library also uses `any`. ☹️ Feel free to dismiss this.